README.md

```diff
@@ -49,5 +49,15 @@ Optional permissions:
 <uses-permission android:name="android.permission.MANAGE_DOCUMENTS"/>
 ```
 
+If you get `NoClassDefFoundError`s because of missing `AppCompatActivity`, enable AndroidX + Jetifier:
```

Comment:
Better "enable AndroidX + Jetifier support in your `android/gradle.properties` file (see [example app](https://github.com/v3rm0n/intercom_flutter/blob/master/android/gradle.properties)):"